recipe/meta.yaml

```diff
@@ -28,12 +28,20 @@ requirements:
     - libtool
     - zlib
     - leptonica
+    - libwebp
+    - xz
+    - openjpeg
+    - giflib
   run:
     - libpng
     - jpeg
     - libtiff
     - zlib
     - leptonica
```

Comment:
Should leptonica be pinned?

Reply:
I defer to your judgment.

Reply:
Looking at https://abi-laboratory.pro/tracker/timeline/leptonica/, we should probably pin to the minor version. Can you add,
```
build:
  run_exports:
    - {{ pin_subpackage("leptonica", max_pin="x.x") }}
```
to leptonica ?

Reply:
https://github.com/conda-forge/leptonica-feedstock/pull/6